docs/source/man.j2

```diff
@@ -189,11 +189,11 @@ Listing and Showing Credentials
 
 The ``{{ QPC_VAR_PROGRAM_NAME }} cred list`` command returns the details for every credential that is configured for {{ QPC_VAR_PROJECT }}. This output includes the name, username, password, SSH keyfile, sudo password, or token (if applicable) for each entry. Passwords and tokens are masked if provided, if not, they will appear as ``null``.
 
-**{{ QPC_VAR_PROGRAM_NAME }} cred list --type=** *(network | vcenter | satellite | openshift | ansible)*
+**{{ QPC_VAR_PROGRAM_NAME }} cred list --type=** *(network | vcenter | satellite | openshift | ansible | acs)*
```

Comment:
just for consistency, maybe have the acs right after openshift like the other changes ?  `openshift | acs | ansible`